### PR TITLE
 Add Qwen Agent World Model to Hub

### PR DESCRIPTION
**Pull request by laxmareddyp**
## Description of the change

Qwen-Agentworld :  https://huggingface.co/collections/Qwen/qwen-agentworld, it is text only language model.

## Qwen-AgentWorld:
 The first language world model to cover seven agent interaction domains within a single model. It simulates agentic environments via long chain-of-thought reasoning, predicting the next environment state given an agent's action and interaction history. Trained through a three-stage pipeline — CPT injects environment knowledge, SFT activates next-state-prediction reasoning, RL sharpens simulation fidelity — Qwen-AgentWorld is a native world model: environment modeling is the training objective from the CPT stage onward, not a post-hoc add-on.

## Highlights
- Seven Unified Domains. A single model covers MCP (tool calling), Search, Terminal, SWE (software engineering), Android, Web, and OS — spanning both text and GUI interaction environments.
- Native World Model. Environment modeling from CPT onward, not post-hoc adaptation on a general-purpose LLM.
- Generalizable, Scalable & Controllable Simulator. Zero-shot generalization to OOD environments (e.g., OpenClaw); controllable perturbations and fictional-world construction surpass real-environment training.
- Agent Foundation Model. LWM RL warm-up on single-turn, non-agentic trajectories transfers to multi-turn, tool-calling agentic tasks across 7 benchmarks, including 3 entirely out-of-domain.

## Colab Notebook
 Numerics verification screenshot:

Screenshot 2026-07-27 at 5 46 45 PM

## Checklist

- [x] I have added all the necessary unit tests for my change.
- [x] I have verified that my change does not break existing code and works with all backends (TensorFlow, JAX, and PyTorch).
- [x] My PR is based on the latest changes of the main branch (if unsure, rebase the code).
- [x] I have followed the Keras Hub [Model contribution guidelines](https://github.com/keras-team/keras-hub/blob/master/CONTRIBUTING_MODELS.md) in making these changes.
- [x] I have followed the Keras Hub [API design guidelines](https://github.com/keras-team/keras-hub/blob/master/API_DESIGN_GUIDE.md) in making these changes.
- [x] I have signed the [Contributor License Agreement](https://cla.developers.google.com/about).

### CODE REVIEW

File: tools/checkpoint_conversion/convert_qwen3_5_moe_checkpoints.py [hunk 1]
```diff
@@ -565,6 +597,14 @@ def save_preset(keras_model, preset_name):
 # ---------------------------------------------------------------
 # Main
 # ---------------------------------------------------------------
+def _is_text_only(hf_preset):
+    """Check if a HuggingFace preset is text-only (no vision weights)."""
+    from transformers import AutoConfig
+
+    config = AutoConfig.from_pretrained(hf_preset)
+    return getattr(config, "language_model_only", False)
```
[the remark this answers is not in the document]
Comment (laxmareddyp) [reply tools/checkpoint_conversion/convert_qwen3_5_moe_checkpoints.py]: Thanks for flagging this. Updated

File: tools/checkpoint_conversion/convert_qwen3_5_moe_checkpoints.py [hunk 2]
```diff
@@ -25,6 +25,7 @@
 from absl import flags
 from keras import ops
 from PIL import Image
+from transformers import AutoModelForCausalLM
 from transformers import AutoModelForImageTextToText
 from transformers import AutoProcessor
 from transformers import AutoTokenizer
```
[the remark this answers is not in the document]
Comment (laxmareddyp) [reply tools/checkpoint_conversion/convert_qwen3_5_moe_checkpoints.py:32]: Thanks for flagging this. Updated

**Review by divyashreepathihalli — APPROVED** [review]
